Glacier Re-Work

WhiteLightningTwister opened the pull request:
Reworking Glaicer, to a more up-to date likewise higher quality standard. Including holopads, refreshed departments and removing confusing or un-synergetic additions. Warp points, beacons proper air alarms systems. Including sweeping changes that drastically increase the viability of an-tags including general players, with whatever they are choosing to do from round to round.

<!--
This is a semi-strict format, you can add/remove sections as needed but the order/format should be kept the same
Remove these comments before submitting
-->

# Current planned changes

<!--
Explain this PR in as much detail as applicable

Some example prompts to consider:
How might this affect the game? The codebase?
What might be some alternatives to this?
How/Who does this benefit/hurt [the game/codebase]?
-->

I am doing sweeping changes to the security, logistics likewise service areas to refine the game play and aesthetics of the station, revamping many under attended parts of the station. Giving it a fresh new coat of plant, allowing more things to happen with revisions of balancing of item location s likewise departmental structures.

---

# TODO

<!--
A list of everything you have to do before this PR is "complete"
You probably won't have to complete everything before merging but it's good to leave future references
-->

- [ ] Ready to push
- [ ] Map Review Approved
- [X] Medical Tweak.
- [X] Epistemics Tweak.
- [X] Security Department Rework.
- [X] Logistics Rework.
- [X] Service revision.
- [X] Command Wing Rework
- [X] AI rework.
- [X] Air alarm systems
- [X] Warp points
- [X] Beacons
- [X] Adding martial artist.
- [X] General map improvements
- [X] Decals & decorations.
- [X] EVAC Check
- [X] Cargo ship Check
- [X] Pathfinder Check
- [X] Gateway To Perma
- [X] Distro & Waste Pipes
- [X] Quad Borg spawn points.
- [X] Added prison guard & martial artist.
- [X] Adjusted crew slots for jobs.

---

<!--
This is default collapsed, readers click to expand it and see all your media
The PR media section can get very large at times, so this is a good way to keep it clean
The title is written using HTML tags
The title must be within the <summary> tags or you won't see it
-->

<details><summary><h1>Media</h1></summary>
<p>

https://cdn.discordapp.com/attachments/1319398540157386823/1442304561028268222/Glacier-0.png?ex=6924f24c&is=6923a0cc&hm=335070bbb7ee7ae3d7b76cb845fe5e933eca88b9e04a83064b3e716a77db2773

</p>
</details>

---

# Changelog

<!--
You can add an author after the `:cl:` to change the name that appears in the changelog (ex: `:cl: Death`)
Leaving it blank will default to your GitHub display name
This includes all available types for the changelog
-->

:cl:
- tweak: Revamped Glacier.

WhiteLightningTwister committed:
Not final. But test push.

The current itteration of the rework of glaicer for observation and review. But not finished yet.

WhiteLightningTwister committed:
PR Review iteration

Finalized the map, found all the errors i could. Added all the things i would think is needed.

WhiteLightningTwister committed:
Crew Slots

Added martial artist, prison guard and adjusted numbers of others. Quad borgs were added too.

WhiteLightningTwister committed:
Fixing Invalid

I think i removed the invalid correctly.

WhiteLightningTwister committed:
Fixed The invalid

Fixing Invalid

Dunrab reviewed (commented):
There are so many issues with this re-work. There are a lot of very un-necessary changes including the sheer size of the map now, logi being expanded massively, salvage being off on its own and being bigger than medical almost. Medical again is left untouched and epi was only very slightly touched. Along with that in my brief time looking at the map its self I found an array of other issues that are to large to post in a single comment so I will be posting them in discord.

WhiteLightningTwister committed:
Changes requested from review

I went about and did the fixes requested by the reviewer. Likewise re-did part of epistemics at their request.

WhiteLightningTwister commented:
We cost host a poll on the discord and see if people are interested in this change. If that would help elevate any concerns.

PubliclyExecutedPig commented:
I do like the station but personally I think the decals on the pathways are too noisy, and kinda repetitive (though that might just be the render not orientating decals correctly)
<img width="247" height="182" alt="image" src="https://github.com/user-attachments/assets/85d93e33-0cd4-4d3a-a122-ec68f0d03b4f" />
